packages/rOpenScPCA/tests/testthat/test-calculate-clusters.R

```diff
@@ -1,9 +1,24 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+suppressPackageStartupMessages(library(Seurat))
+
+sce <- scpcaTools:::sim_sce(n_cells = 100, n_genes = 10, n_empty = 0)
 test_mat <- matrix(
   runif(1000, -3, 3),
   nrow = 100,
   ncol = 10
 )
```

Comment:
At this point you might as well get the real PCA matrix from the SCE?

Reply:
I did this with a very quick 'n dirty normalization and `runPCA`, which does add `scater` to Suggests. Is this what you had in mind?